lib/inc/Recorder.h

```diff
@@ -290,6 +290,18 @@ namespace sairedis
                     _In_ sai_status_t status,
                     _In_ const uint64_t *count);
 
+            void recordQueryAattributeCapability(
+                    _In_ sai_object_id_t switch_id,
+                    _In_ sai_object_type_t object_type,
+                    _In_ sai_attr_id_t attr_id,
+                    _Out_ sai_attr_capability_t *capability);
+
+            void recordQueryAattributeCapabilityResponse(
```

Comment:
double "Aa" in name

Reply:
Removed the extra 'a" from function name.